src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -0,0 +1,359 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2023 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+package org.isf.medicalinventory.gui;
+
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.GregorianCalendar;
+
+import javax.swing.JButton;
+import javax.swing.JComboBox;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTable;
+import javax.swing.SwingConstants;
+import javax.swing.border.EmptyBorder;
+import javax.swing.table.DefaultTableModel;
+
+import org.isf.generaldata.MessageBundle;
+import org.isf.utils.jobjects.GoodDateChooser;
+import org.isf.utils.jobjects.InventoryState;
+import org.isf.utils.jobjects.ModalJFrame;
+import org.isf.utils.time.TimeTools;
+
+public class InventoryWardBrowser extends ModalJFrame {
+
+    private static final long serialVersionUID = 1L;
+
+    private GoodDateChooser jCalendarTo;
+    private GoodDateChooser jCalendarFrom;
+    private LocalDateTime dateFrom = TimeTools.getDateToday0();
+    private LocalDateTime dateTo = TimeTools.getDateToday24();
+    private JLabel jLabelTo;
+    private JLabel jLabelFrom;
+    private JPanel panelHeader;
+    private JPanel panelFooter;
+    private JPanel panelContent;
+    private JButton closeButton;
+    private JButton newButton;
+    private JButton updateButton;
+    private JButton deleteButton;
+    private JButton viewButton;
+    private JScrollPane scrollPaneInventory;
+    private JTable jTableInventory;
+    private String[] pColums = { MessageBundle.getMessage("angal.inventory.referenceshow.col"),
```

Comment:
Column labels are uppercased; see for example: https://github.com/informatici/openhospital-gui/blob/8c885df8a52cc36e79a08e2fa4627efe40060116/src/main/java/org/isf/admission/gui/PatientDataBrowser.java#L197